bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/repository/LogisticAddressRepository.kt

```diff
@@ -43,6 +43,9 @@ interface LogisticAddressRepository : PagingAndSortingRepository<LogisticAddress
         pageable: Pageable
     ): Page<LogisticAddress>
 
+    @Query("SELECT a FROM LogisticAddress a WHERE LOWER(a.name) LIKE LOWER(CONCAT('%', :addressName, '%')) ORDER BY LENGTH(a.name)")
+    fun findByName(addressName: String, pageable: Pageable): Page<LogisticAddress>
```

Comment:
In order to increase performance please concatenate the search string before passing it to the repository method.

Reply:
I checked with Martin regarding this, its just one Concat which will not reduce much performance.
Still adapted changes requested.

Reply:
I don't think a concat on a search parameter has a noticeable performance impact because it needs to be performed only once for each query. Doing a concat on a DB field is quite another thing and should be avoided.

Reply:
I was not aware that the parameter is only concatenated once before it enters the query. I guess that makes sense. If that's really the case then I don't see any problem with this